### PR TITLE
docs(tutorials): review content 20220811 MTA-2132

### PR DESCRIPTION
**Pull request by bene2k1**
### Description
Tutorial review 2022-08-11

### CODE REVIEW

File: tutorials/deploy-ubuntu-20-04-instance-scaleway-elements/index.mdx [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
   - compute
 dates:
-  validation: 2022-02-03
-  posted: 2021-12-02
+  validation: 2022-08-11
+  posted: 2022-08-11
 ---
 
-[Ubuntu](https://ubuntu.com/) is one of the most popular Linux distributions. 
+Ubunt uis one of the most popular Linux distributions.
```
Comment (RoRoJ) [inline tutorials/deploy-ubuntu-20-04-instance-scaleway-elements/index.mdx:16]:
```suggestion
Ubunt is one of the most popular Linux distributions. 
```

File: tutorials/deploy-ubuntu-22-04-instance/index.mdx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    adduser scaler
+    ```
+2. Answer the questions, starting with the account password. Make sure to use a strong password and optionally fill in any of the other information. Confirm the response to each question by pressing Enter.
+
+3. Add the users to the **superdoers** group (sudo) to be able to launch administrative tasks from this user, without the need to log in as `root`.
+    ```
+    adduser scaler sudo
+    ```
+    You can now run commands with administrative permissions by prefixing your command with `sudo`. For example: `sudo apt upgrade`.
+3. Add the public SSH key information for the user by opening the file `~/.ssh/authorized_keys` in a text editor and pasting the public key:
```
Comment (RoRoJ) [inline tutorials/deploy-ubuntu-22-04-instance/index.mdx:118]: Step 3 twice, and no linebreak between steps

File: tutorials/deploy-ubuntu-20-04-instance-scaleway-elements/index.mdx [hunk 3]
```diff
@@ -40,11 +40,11 @@ In this tutorial you will learn how to deploy an Instance with Ubuntu 20.04 LTS
     * Select the Instance type
         * Scaleway provide different Instance types, designed for specific use cases. Choose the configuration that fits your requirements.
     * Choose an image for your instance
```
Comment (nedaehm) [inline tutorials/deploy-ubuntu-20-04-instance-scaleway-elements/index.mdx:42]:
```suggestion
    * Choose an image for your Instance
```

File: tutorials/deploy-ubuntu-22-04-instance/index.mdx [hunk 4]
```diff
@@ -0,0 +1,126 @@
+---
+meta:
+  title: How to deploy a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway
```
Comment (nedaehm) [inline tutorials/deploy-ubuntu-22-04-instance/index.mdx:3]:
```suggestion
  title: How to deploy an Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway
```

File: tutorials/deploy-ubuntu-22-04-instance/index.mdx [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+  description: In this tutorial you will learn how to deploy, update and manage a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway.
```
Comment (nedaehm) [inline tutorials/deploy-ubuntu-22-04-instance/index.mdx:4]:
```suggestion
  description: In this tutorial you will learn how to deploy, update and manage an Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway.
```

File: tutorials/deploy-ubuntu-22-04-instance/index.mdx [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+content:
+  h1: How to deploy a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway
```
Comment (nedaehm) [inline tutorials/deploy-ubuntu-22-04-instance/index.mdx:6]:
```suggestion
  h1: How to deploy an Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway
```

File: tutorials/deploy-ubuntu-22-04-instance/index.mdx [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+  paragraph: In this tutorial you will learn how to deploy a Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway.
```
Comment (nedaehm) [inline tutorials/deploy-ubuntu-22-04-instance/index.mdx:7]:
```suggestion
  paragraph: In this tutorial you will learn how to deploy an Ubuntu 20.04 LTS (Focal Fossa) Instance on Scaleway.
```

**Review by nedaehm — APPROVED** [review]

**Pull request merged**